src/html/main.html

```diff
@@ -67,6 +67,7 @@
                         <div class="grade">Grade</div>
                         <div class="solutions">Solutions</div>
                         <div class="submissions">Submissions</div>
+                        <div class="comments">Comments (Hover)</div>
```

Comment:
Consider removing ` (Hover)`?  Especially after I thought of adding the `class="link"`, it might be obvious from the UI that you're supposed to hover on them.